packages/block-editor/src/components/inserter/index.js

```diff
@@ -139,7 +142,10 @@ class Inserter extends Component {
 		return (
 			<Dropdown
 				className="block-editor-inserter"
-				contentClassName="block-editor-inserter__popover"
+				contentClassName={ classnames(
+					'block-editor-inserter__popover',
+					{ 'is-top-toolbar-inserter': ! isAppender }
```

Comment:
This variable used to distinguish the visual styling between the top toolbar inserter and the one invoked from the sibling inserter. In a rebase, I failed to verify this still worked before pushing, and somehow it broke. I don't know why it doesn't work anymore. @youknowriad if you have time, can you provide insights into this?

Here's how it looks when broken.

<img width="1097" alt="Screenshot 2020-02-03 at 10 08 37" src="https://user-images.githubusercontent.com/1204802/73639881-8847ae00-466d-11ea-8d84-2c6f4efa27da.png">

Reply:
This is definitely not the right variable to check for and in fact we don't have any variable to distinguish these properly at the moment.

I think we follow the CSS suggestions I propose (avoid the large width and use absolute positionning for the preview), we could show the same inserter in all places and it will be shown properly with less code. We might need to add a dedicated prop to remove the preview `hideBlockPreview` or somthing but the overall design will stay consistent.

Reply:
To clarify the issue ini that screenshot is that the width of the popover is very large (not just the panel on the left) so centering the toggle appear broken. Absolute positioning of the preview fixes that.